dts/bindings/sensor/st,lsm9ds1_mag.yaml

```diff
@@ -0,0 +1,83 @@
+description: |
+    STMicroelectronics LSM9DS1 9-axis IMU (Inertial Measurement Unit) sensor
+    accessed through I2C bus.
+
+    This binding describe only the magnetometer.
+
+    When setting the mag-range and mag-odr properties in
+    a .dts or .dtsi file you may include lsm9ds1.h and use the macros
+    defined here.
+
+    Example:
+    #include <zephyr/dt-bindings/sensor/lsm9ds1.h>
+
+    lsm9ds1_mag: lsm9ds1_mag@0 {
+      ...
+
+      mag-range = <LSM9DS1_DT_FS_4Ga>;
+      mag-odr = <LSM9DS1_MAG_MP_40Hz>;
+    };
+
+compatible: "st,lsm9ds1_mag"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  mag-range:
+    type: int
+    default: 0
+    description: |
+      Range of the magnetometer in gauss. Default is power-up configuration.
+
+      - 0 # LSM9DS1_DT_FS_4Ga   (0.14  mgauss/LSB)
+      - 1 # LSM9DS1_DT_FS_8Ga   (0.29  mgauss/LSB)
+      - 2 # LSM9DS1_DT_FS_12Ga  (0.43  mgauss/LSB)
+      - 3 # LSM9DS1_DT_FS_16Ga  (0.58  mgauss/LSB)
+
+    enum: [0,1,2,3]
+
+  mag-odr:
+    type: int
+    default: 0
```

Comment:
missing justification

Reply:
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values